cmd/helm/helm.go

```diff
@@ -33,11 +33,15 @@ import (
 
 	"helm.sh/helm/pkg/action"
 	"helm.sh/helm/pkg/cli"
+	"helm.sh/helm/pkg/gates"
 	"helm.sh/helm/pkg/kube"
 	"helm.sh/helm/pkg/storage"
 	"helm.sh/helm/pkg/storage/driver"
 )
 
+// FeatureGateOCI is the feature gate for checking if `helm chart` and `helm registry` commands should work
+const FeatureGateOCI = gates.Gate("HELM_EXPERIMENTAL_OCI")
```

Comment:
should this be public?

Reply:
this is helm-specific code so I'd prefer to keep this in `cmd` as an internal variable for the CLI.

This should be exposed as part of #5311 though.